eslint/_base.js

```diff
@@ -12,12 +12,12 @@ module.exports = {
     'plugin:import/recommended',
     'prettier',
     require.resolve('./rules/best-practice'),
-    require.resolve('./rules/comments'),
```

Comment:
I added a comments config and a unicorn config to ensure that only pure rules, excluding plugins, are in the rules directory.